pkg/security/secl/model/field_handlers_others.go

```diff
@@ -1,8 +1,11 @@
+//go:build !linux
```

Comment:
Am I understanding the change here that now the MacOS build is tied to the Windows build?
Which means that as we add to the windows implementation, the mac build/test will compile the windows code?

Reply:
No real platform specific code should go there, the platform specific code should go in the probe package and not in secl/model

Reply:
except there already is platform specific code which is why we have `field_handlers_linux` and now `field_handlers_!linux`
I think this is because the fields themselves can be labelled with platform tags?

given that, I'd prefer to not have a situation (again) where changing windows code could break the mac tests

Reply:
What was currently available in field_handler was not directly used for evaluation but for our tests, as a way to backfill events from a json file in order to run tests